dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/impl/AwsClient.scala

```diff
@@ -94,6 +97,31 @@ private[dynamodb] trait AwsClient[S <: AwsClientSettings] {
       .withAttributes(ActorAttributes.supervisionStrategy(decider))
       .map(_.asInstanceOf[Op#B])
 
+  def flowOp[Op <: AwsOp](op: Op): Flow[Op, Op#B, NotUsed] = {
+    val opFlow =
+      Flow[Op]
+        .map(op => toAwsRequest(op))
+        .via(connection)
+        .mapAsync(settings.parallelism) {
+          case (Success(response), awsRequestMeta) => toAwsResult(response, awsRequestMeta)
+          case (Failure(ex), _) => Future.failed(ex)
+        }
+
+    opFlow
+      .retryWithBackoff(
+        settings.retrySettings.maximumAttempts,
+        settings.retrySettings.initialRetryTimeout,
+        settings.retrySettings.backoffStrategy, {
+          case _ @(_: InternalServerErrorException | _: ItemCollectionSizeLimitExceededException |
+              _: LimitExceededException | _: ProvisionedThroughputExceededException |
+              _: RequestLimitExceededException) =>
```

Comment:
It is possible to set supervision strategy on a stream operator as a stream attribute. We should use that instead so users can customize supervision strategies: https://doc.akka.io/docs/akka/current/stream/stream-error.html#supervision-strategies